### DISCUSSION TITLE
自助分析问题清单

### DISCUSSION THREAD

**Discussion by Esorakouki** [antvis/S2#480] [Q&A]
--------------高优 ---------------
 - [ ] #868 
 - [ ] 树形表卡死：https://teambition.alibaba-inc.com/task/61af08b796e73303931d518a  (不确定是不是表的问题 暂时没提TB)
 - [ ] #873 
 - [ ] #879 

--------------普优----------------

**Comment by xingwanying** [comment]
可否加个issue解决的预期时间

**Reply by Esorakouki** [reply]
> 可否加个issue解决的预期时间

就上周五定的这周五（预期

**Comment by BBSQQ** [comment]
最高优+1: https://github.com/antvis/S2/issues/535

**Comment by BBSQQ** [comment]
高优+1: https://github.com/antvis/S2/issues/565 
期望周四可以都清掉🥺
